### PR TITLE
CSM-68 Exclude transitive dependencies

### PR DESCRIPTION
**Pull request by rkorytkowski**
https://issues.openmrs.org/browse/CSM-68

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -19,103 +19,12 @@
 			<groupId>org.other.library</groupId> 
 			<artifactId>library-name</artifactId>
 		 </dependency> -->
+		 
 		<dependency>
 			<groupId>${project.parent.groupId}</groupId>
 			<artifactId>${project.parent.artifactId}-server</artifactId>
 			<version>${project.parent.version}</version>
 		</dependency>
-
```
Comment (rkorytkowski) [inline api/pom.xml]: All the following are defined in the main pom. No need to repeat.

File: omod/pom.xml [hunk 2]
```diff
@@ -17,86 +17,36 @@
...
-		<!-- Begin OpenMRS modules -->
-
 		<dependency>
 			<groupId>${project.parent.groupId}</groupId>
 			<artifactId>${project.parent.artifactId}-api</artifactId>
 			<version>${project.parent.version}</version>
 		</dependency>
-
 		
-		<!-- End OpenMRS modules -->
```
Comment (rkorytkowski) [inline omod/pom.xml]: Removed dependencies already defined in the main pom.

File: pom.xml [hunk 3]
```diff
@@ -112,46 +110,156 @@
...
+				<version>1.9.10</version>
+			</dependency>
+
+			<dependency>
+				<groupId>net.sf.json-lib</groupId>
+				<artifactId>json-lib</artifactId>
+				<version>2.4</version>
+				<classifier>jdk15</classifier>
+				<exclusions>
+					<exclusion>
```
Comment (rkorytkowski) [inline pom.xml]: Added exclusions for dependencies already available in core.

File: api/pom.xml [hunk 4]
```diff
@@ -19,103 +19,12 @@
...
-			<version>4.3.1</version>
-		</dependency>
-
-		<!-- End Solr -->
-		
-		<dependency>
-			<groupId>org.slf4j</groupId>
-			<artifactId>slf4j-simple</artifactId>
-			<version>1.6.2</version>
-		</dependency>
```
Comment (rkorytkowski) [inline api/pom.xml]: Removed this entirely as it didn't seem to be used.

**Comment by dkayiwa** [comment]
@rkorytkowski i still see commons-logging in the lib folder after compiling it. Did you intend to leave it therein? :)

**Comment by rkorytkowski** [comment]
@dkayiwa we don't have commons-logging in core so that one should still be there. I don't see log4j nor slf4j on my machine in omod's lib directory. Not sure why it's different for you. Could you please try building again? If it's still there, please pastebin the output from mvn dependency:tree.

**Comment by dkayiwa** [comment]
@rkorytkowski you are right. I think something was just wrong with my setup. I merged and pulled again to confirm that all is well now. :)

**Pull request merged**